data_loss.rst

```diff
@@ -473,3 +474,30 @@ The following command can be used to monitor the progress of any ongoing balance
 .. code-block:: console
 
     btrfs balance status /mnt2/pool-name
+
+.. _rockonrestore:
+
+Rock-on Restore
+^^^^^^^^^^^^^^^
+
+When Rock-ons are not visible after data restore it is possible that they are either running silently in the background
+or Rockstor does not have the needed information to restore the Rock-ons. Here is how to get your Rock-ons back up and
+running.
+
+Rock-on data was not stored on system drive (recommended)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Restore your data, pools and shares to make sure the Rock-on data is available
+2. Restore your Rockstor configuration
+
+This should be enough for Rockstor to restore your Rock-ons and make them visible and manageable from the UI again.
+
+
+Rock-on data stored on system drive (discouraged)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Restore your Rock-on data and the needed shares of your system drive to make sure the Rock-on data is available again (this would be the ideal moment to move the data to a share that is not on the system drive!)
```

Comment:
```suggestion
1. Restore your Rock-on associated shares on your system drive to make sure they are available again (this would be the ideal moment to instead restore/move them to a share that is not on the system drive!)
```

Reword in accordance with the title changes earlier.